docs/pages/web/banneroverlay.js

```diff
@@ -0,0 +1,206 @@
+// @flow strict
+import { type Node as ReactNode } from 'react';
+import AccessibilitySection from '../../docs-components/AccessibilitySection';
+import docGen, { type DocGen } from '../../docs-components/docgen';
+import GeneratedPropTable from '../../docs-components/GeneratedPropTable';
+import MainSection from '../../docs-components/MainSection';
+import Page from '../../docs-components/Page';
+import PageHeader from '../../docs-components/PageHeader';
+import SandpackExample from '../../docs-components/SandpackExample';
+import avatar from '../../examples/banneroverlay/avatar';
+import desktop from '../../examples/banneroverlay/desktop';
+import doConcise from '../../examples/banneroverlay/doConcise';
+import doEducate from '../../examples/banneroverlay/doEducate';
+import doNavigate from '../../examples/banneroverlay/doNavigate';
+import dontCritical from '../../examples/banneroverlay/dontCritical';
+import dontLong from '../../examples/banneroverlay/dontLong';
+import dontStack from '../../examples/banneroverlay/dontStack';
+import icon from '../../examples/banneroverlay/icon';
+import image from '../../examples/banneroverlay/image';
+import main from '../../examples/banneroverlay/main';
+import mobile from '../../examples/banneroverlay/mobile';
+
+export default function DocsPage({ generatedDocGen }: { generatedDocGen: DocGen }): ReactNode {
+  return (
+    <Page title={generatedDocGen?.displayName}>
+      <PageHeader
+        name={generatedDocGen?.displayName}
+        description={generatedDocGen?.description}
+        pdocsLink
+      >
+        <SandpackExample code={main} name="Main BannerOverlay example" hideEditor />
+      </PageHeader>
+      <GeneratedPropTable generatedDocGen={generatedDocGen} />
+      <MainSection name="Usage guidelines">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            title="When to use"
+            description={`
+- To provide short educational messages allowing content to scroll underneath
+- To help users learn more about a specific idea or interest
+- To support users when they have performed actions that indicate medium or high intent (related pin tap, idea pin swipe)
+        `}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            title="When not to use"
+            description={`
+- If there is a need to block the content underneath Use Modal instead
+- To replace Toast
+- For lengthy messages, forms, or blocks of information. Consider OverlayPanel or new page instead
+        `}
+          />
+        </MainSection.Subsection>
+      </MainSection>
+      <MainSection name="Best Practices">
+        <MainSection.Subsection columns={1}>
```

Comment:
This should be 2 columns right? Like our other docs wdyt

Reply:
Or is for size wrapping it's like this?

Reply:
Yeah, I set it as 1 column because the text will wrap differently from actual dialog usage if cramped in columns. Similar to SlimBanner examples.